tasks/IWantThis.tex

```diff
@@ -0,0 +1,91 @@
+\section{I Want This [Party Host]}
+A guest at the party speaks English, but with only a limited vocabulary. They want a drink, an hors d'oeuvre, and a set of utensils, but do not know the words to describe them. As such, they will ask the robot for one through gesturing, and will ask for recommendations made by the robot through gestures. They will also discuss a picture on the wall, which the robot will determine by analyzing their gaze.
```

Comment:
To Do: Shrink description.

Whether this is approved or not, I think we have two tasks here, one for retrieving objects (gesture-detection-based HRI) and another for discussing the painting (dialogue-based HRI).